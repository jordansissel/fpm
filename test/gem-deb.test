#!/bin/bash
RAILS_VERSION=3.1.0

run() {
  # Force the gem installation path to match the one of the expected output
  export GEM_HOME=/usr/lib/ruby/gems/1.8
  
  fpm -s gem -t deb -v $RAILS_VERSION rails

  file=rubygem-rails_${RAILS_VERSION}_all.deb
  dpkg -c $file | awk '{$1=$2=$3=$4=$5="";print}' | sed -e 's/^ *//' | sort > $output
  rm $file
}

clean() {
  rm rails-${RAILS_VERSION}.gem
}
