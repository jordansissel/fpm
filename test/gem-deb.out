./usr/
./usr/lib/
./usr/lib/ruby/
./usr/lib/ruby/gems/
./usr/lib/ruby/gems/1.8/
./usr/lib/ruby/gems/1.8/bin/
./usr/lib/ruby/gems/1.8/bin/rails
./usr/lib/ruby/gems/1.8/cache/
./usr/lib/ruby/gems/1.8/cache/rails-3.1.0.gem
./usr/lib/ruby/gems/1.8/doc/
./usr/lib/ruby/gems/1.8/gems/
./usr/lib/ruby/gems/1.8/gems/rails-3.1.0/
./usr/lib/ruby/gems/1.8/gems/rails-3.1.0/bin/
./usr/lib/ruby/gems/1.8/gems/rails-3.1.0/bin/rails
./usr/lib/ruby/gems/1.8/specifications/
./usr/lib/ruby/gems/1.8/specifications/rails-3.1.0.gemspec
